JustSaying/AwsTools/MessageHandling/SqsNotificationListener.cs

```diff
@@ -86,22 +85,16 @@ public SqsNotificationListener WithMessageProcessingStrategy(IMessageProcessingS
             _handlerMap.Add(typeof(T), handlerFunc);
         }
 
-        public void Listen()
+        public void Listen(CancellationToken cancellationToken = default)
```

Comment:
I agree with @AnthonySteele we don't need an optional parameter for the Listen method, because we have handled it in the Start method. I think I suggested it earlier not realising we we're in the "inner call"

Reply:
Done